src/embind/embind_ts.js

```diff
@@ -210,6 +210,13 @@ var LibraryEmbind = {
         ['bool', 'boolean'],
         ['float', 'number'],
         ['double', 'number'],
+#if WASM_BIGINT
+        ['int64_t', 'bigint'],
+        ['uint64_t', 'bigint'],
+#else
```

Comment:
IIRC, if bigint is not enabled, won't using a `int64_t` cause a runtime type error? I'm thinking we should not emit a map for these values if they'll be broken anyway.

Reply:
@brendandahl not sure if I'm getting something wrong, but according to the FAQ when the `WASM_BIGINT` flag is not provided the int64 values will be legalized to standard 32 bit numbers, and the missing higher bits can be accessed by using the `getTempRet0` helper

Reply:
I updated the code so that the mapping is only enabled when `WASM_BIGINT` flag is present, failing otherwise. I also updated the test and the docs. Let me know if everything looks good now.